plotly/tools.py

```diff
@@ -2871,3 +2891,149 @@ def get_candle_decrease(self):
         return (decrease_x, decrease_close, decrease_dif,
                 stick_decrease_y, stick_decrease_x)
 
+class _Dendrogram(FigureFactory):
+    ''' Returns a Dendrogram figure object
+        Example usage:
+        D = Dendrogram( Z )
+        fig = { 'data':D.data, 'layout':D.layout }
+        py.iplot( fig, filename='Dendro', validate=False )'''
+    
+    def __init__(self, X, orientation='bottom', labels=None, colorscale=None, \
+                 width=700, height=700, xaxis='xaxis', yaxis='yaxis' ):
+        ''' Draw a 2d dendrogram tree 
```

Comment:
We've been adding things like this with a `BETA` tag in the docstring. This way we don't have to promise backwards compatibility on these new wrappers. We simply use:

```
"""
Draw a 2d dendrogram tree.

BETA: May change without warning in a backwards incompatible way.

...
"""
```